packages/cli/create-strapi-app/templates/example-js/package.json

```diff
@@ -1,10 +1,23 @@
 {
   "scripts": {
+    "build": "strapi build",
+    "deploy": "strapi deploy",
     "develop": "strapi develop",
+    "dev": "strapi develop",
+    "console": "strapi console",
     "start": "strapi start",
-    "build": "strapi build",
     "strapi": "strapi",
-    "deploy": "strapi deploy",
+    "strapi:routes": "strapi routes:list",
+    "strapi:policies": "strapi policies:list",
+    "strapi:middlewares": "strapi middlewares:list",
+    "strapi:content-types": "strapi content-types:list",
+    "strapi:controllers": "strapi controllers:list",
+    "strapi:services": "strapi services:list",
+    "upgrade": "npx @strapi/upgrade latest",
+    "upgrade:dry": "npx @strapi/upgrade latest --dry",
+    "upgrade:major": "npx @strapi/upgrade major",
+    "upgrade:minor": "npx @strapi/upgrade minor",
+    "upgrade:patch": "npx @strapi/upgrade patch",
```

Comment:
you can have only the upgrade (and maybe the dry run ) one and people will be able to do `yarn upgrade minor` which seems more consistent with the actual command. Let's avoid creating confusion with the documentation